gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
```

Comment:
Ok, I renamed it. I already have observatory for observatory name. As for other parameters, let's see if they are needed, and we can add them later.